packages/step-wizard/WizardStepBadge.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const WizardStepBadge = ({ tag: Tag, className: classes, children }) => (
+  <Tag className={classNames(classes, 'stepwizard-badge')}>{children}</Tag>
```

Comment:
Same thing with the badge. Better now than have it implemented later.

Reply:
163fdf01c59f2a5b5015f765064b27c2d29e9e37